WTWSMS/history/titles/c_labourd.txt

```diff
@@ -38,9 +38,13 @@
 561.1.1={
 	holder = 206851 # Charibert Meroving 
 }
+565.1.1={
+	holder = 1310013 # Nicetius
+	liege = "k_paris"
+}
 567.1.1={
+    liege = "k_neustria"
+}
+581.1.1={
 	holder = 30021 # Chilperic Meroving 
 }
-584.1.1={
-	holder = 30029 # Chlothar II Meroving 
-}
```

Comment:
Why did you remove this?

Reply:
IDK, I may have deleted it unconciously. I´m taking it back.

Reply:
Okay.